tests/templates/loops.html

```diff
@@ -11,8 +11,8 @@
       {% if number_reviews > 10 or show_more %}
         <p>All reviews</p>
         {% for review in reviews %}
-          <h3>{{review.title}}</h3>
-          {% for paragraph in review.paragraphs %}
+          <h3>{{review.title}}</h3>{% set rev = review %}
```

Comment:
can you create a new test for that? otherwise I'm never going to remember why it's there :)
If you can't, no worries i'll do it myself after mergin